documentation/puppet/v3/environment_classes.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Thanks for the suggestion.  I used that site to validate my sample json response against the schema in the PR and it was successful.  The schema seems to at least be syntactically valid, then.